src/main/kotlin/racing/view/OutputView.kt

```diff
@@ -1,19 +1,24 @@
 package racing.view
 
-class OutputView {
+import racing.domain.Car
+
+private const val POSITION_CHAR = "-"
+
+internal class OutputView {
 
     fun renderStartMessage() {
-        print("실행 결과")
+        println("\n실행 결과")
     }
 
-    fun renderPositions(positions: List<Int>) {
-        println()
-        positions.forEach {
-            val builder = StringBuilder()
-            for (i in 0 until it) {
-                builder.append('-')
-            }
-            println(builder.toString())
+    fun renderRound(cars: List<Car>) {
+        cars.forEach {
+            println("${it.name} : ${POSITION_CHAR.repeat(it.position)}")
         }
+        println()
+    }
+
+    fun renderWinners(winners: List<Car>) {
+        val winnerNames = winners.joinToString(separator = ", ") { car -> car.name }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val winnerNames = winners.joinToString { it.name }
```